src/lavinmq/mfile.cr

```diff
@@ -185,6 +185,14 @@ class MFile < IO
     @size = new_size
   end
 
+  def write_at(pos : Int64, slice : Bytes) : Nil
+    raise ClosedError.new if @closed
+    end_pos = pos + slice.size
+    raise IO::EOFError.new if end_pos > @capacity
+    @size = end_pos if end_pos > @size
+    slice.copy_to(buffer + pos, slice.size)
+  end
+
```

Comment:
Can't see that this is being used? Am I missing something?

```suggestion
```

Reply:
Yes, you are right. We refactored away from that earlier. Will remove it!